data_steward/validation/main.py

```diff
@@ -274,33 +274,68 @@ def validate_submission(hpo_id: str, bucket, folder_items: list,
 
     # Create all tables first to simplify downstream processes
     # (e.g. ehr_union doesn't have to check if tables exist)
-    for file_name in resources.CDM_CSV_FILES + common.PII_FILES:
+    expected_tables = []
+    for file_name in resources.CDM_CSV_FILES + common.PII_CSV_FILES:
         table_name = file_name.split('.')[0]
         table_id = resources.get_table_id(table_name, hpo_id=hpo_id)
+        expected_tables.append(table_id)
         bq_utils.create_standard_table(table_name, table_id, drop_existing=True)
 
-    for cdm_file_name in sorted(resources.CDM_CSV_FILES):
-        file_results, file_errors = perform_validation_on_file(
-            cdm_file_name, found_cdm_files, hpo_id, folder_prefix, bucket)
-        results.extend(file_results)
-        errors.extend(file_errors)
+    found_tables = []
+    found_csv_files, found_parquet_files, found_jsonl_files = [], [], []
+    found_csv_tables, found_parquet_tables, found_jsonl_tables = [], [], []
+    for found_file in found_cdm_files + found_pii_files:
+        if found_file.endswith('.csv'):
+            found_csv_files.append(found_file)
+            found_csv_tables.append(found_file.split('.')[0])
+        elif found_file.endswith('.parquet'):
+            found_parquet_files.append(found_file)
+            found_parquet_tables.append(found_file.split('.')[0])
+        elif found_file.endswith('.jsonl'):
+            found_jsonl_files.append(found_file)
+            found_jsonl_tables.append(found_file.split('.')[0])
+        else:
+            logging.info(f"Ignoring unexpected file type: {found_file}")
+
+    # Remove any csv files that have corresponding parquet or jsonl files
+    found_csv_files = [
```

Comment:
So, what you're doing here is prioritizing parquet > jsonl > csv, right?

Reply:
Yes that's correct